### DISCUSSION TITLE
react约定路由如何取消导航和左侧菜单

### DISCUSSION THREAD

**Discussion by gjmhandsome** [umijs/dumi#1942] [Q&A]
请教一下，在.dumi/pages下写的react页面，如何只生成路由，而不展示导航和左侧菜单

**Comment by PeachScript** [comment]
Duplicate of https://github.com/umijs/dumi/discussions/1936

**Comment by PeachScript** [comment]
导航无法取消，因为约定式 React 页面的使用场景是在现有文档中添加自定义的页面；sidebar [可以配置关闭](https://d.umijs.org/config/markdown#sidebar)，Markdown 的 FrontMatter 在约定式 React 页面组件头部也是可用的：

```js
/**
 * key: value
 */

// 其他代码...
```

**Comment by nbbjack** [comment]
你们真是个大机灵鬼
